Assets/Prefabs/Player/Weapons/ShotGun.prefab

```diff
@@ -10,7 +10,7 @@ GameObject:
   m_Component:
   - component: {fileID: 479315117325413788}
   m_Layer: 0
-  m_Name: Gun_No_Round 1
+  m_Name: ShotGun
```

Comment:
```suggestion
  m_Name: Shotgun
```
I've made this change in my branch [feature/rls-7-shotgun-behaviour](https://github.com/JamesSchiele/DungeonGenerator/tree/feature/rls-7-shotgun-behaviour) so you should be able to rebase off that and get this for free